streams/src/test/java/org/apache/kafka/streams/processor/internals/ActiveTaskCreatorTest.java

```diff
@@ -141,118 +254,613 @@ public void shouldFailForUnknownTaskOnStreamsProducerPerTask() {
         {
             final IllegalStateException thrown = assertThrows(
                 IllegalStateException.class,
-                () -> activeTaskCreator.streamsProducerForTask(new TaskId(0, 0))
+                () -> activeTaskCreator.streamsProducerForTask(new TaskId(0, 2))
             );
 
-            assertThat(thrown.getMessage(), is("Unknown TaskId: 0_0"));
+            assertThat(thrown.getMessage(), is("Unknown TaskId: 0_2"));
         }
     }
 
     @Test
-    public void shouldReturnStreamsProducerPerTask() {
```

Comment:
Test moved

Reply:
It reminds me that sometimes I did the same thing of piggy-backing re-orging in the code base with actual changes that makes the reviewers life harder :P All of us should be careful in the future.

That being said, for this PR since Boyang and I have bite it already, I think we can save reverting the piggy-backed reordering and just merge as this.

Reply:
I agree and I usually try to avoid re-ordering. However, because I added all missing test method it was required to keep an overview and to ensure we don't miss a case... Sorry for the mess.